crypto/fipsmodule/ecdsa/make_ecdsa_p192_secp256k1_tests.go

```diff
@@ -0,0 +1,281 @@
+/* Copyright (c) 2018, Google Inc.
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
+ * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+ * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
+
+package main
+
+import (
+  "crypto/aes"
+  "crypto/cipher"
+  "crypto/ecdsa"
+  "crypto/elliptic"
+  "crypto/rand"
+  "crypto/sha1"
+  "crypto/sha256"
+  "crypto/sha512"
+  "math/big"
+  mathrand "math/rand"
+  "hash"
+  "fmt"
+  "io"
+  "os"
+)
+
+import "github.com/ethereum/go-ethereum/crypto/secp256k1"
+
+// START - Deterministic RNG helper functions.
```

Comment:
It's awesome that you introduced a go script that can do the deterministic RNG k generation 👏 .